src/shared/contexts/query/context.tsx

```diff
@@ -27,23 +28,26 @@ import {notify} from 'src/shared/actions/notifications'
 import {resultTooLarge} from 'src/shared/copy/notifications'
 
 // Types
-import {CancellationError} from 'src/types'
+import {CancellationError, OwnBucket} from 'src/types'
 import {RunQueryResult} from 'src/shared/apis/query'
 import {event} from 'src/cloud/utils/reporting'
 import {PROJECT_NAME} from 'src/flows'
+import {LanguageType} from 'src/dataExplorer/components/resources'
 
 interface CancelMap {
   [key: string]: () => void
 }
 
 export enum OverrideMechanism {
-  Inline,
-  AST,
-  JSON,
+  Inline = 'inject-variables-into-script',
+  Extern = 'AST',
+  Params = 'JSON', // TODO: code is incorrect & will need debugging
```

Comment:
Made these names a bit more explicit.
enum is `<ApproachName> = '<payloadShape>'`